pkg/cluster/volumes.go

```diff
@@ -184,21 +191,53 @@ func (c *Cluster) populateVolumeMetaData() error {
 }
 
 // syncVolumeClaims reads all persistent volume claims and checks that their size matches the one declared in the statefulset.
-func (c *Cluster) syncVolumeClaims() error {
+func (c *Cluster) syncVolumeClaims(noResize bool) error {
```

Comment:
I tend to call these negative variable ignoreSomething. I my head it's easier to process the following code, e.g. I need a few seconds more on !noResize